profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestCatalog.java

```diff
@@ -292,4 +294,27 @@ public void testGetCatalogJSONWithRecurringBillingMode() throws Exception {
         Assert.assertNotEquals(catalogsJson.get(0).getProducts().get(2).getPlans().get(0).getRecurringBillingMode(), null);
 
     }
+
+//    @Test(groups = "slow")
+//    public void testValidateCatalog() throws Exception {
```

Comment:
Cleanup?

Reply:
Had commented out this test since there were some merge issues with the `killbill-client-java` repo which caused compilation errors in this test. The merge issues are now resolved so I've uncommented the test.